src/list.coffee

```diff
@@ -81,11 +83,11 @@ class List extends Command
       manifest ?= {}
       manifest.name = child
       if options.argv.themes
-        packages.push(manifest) if manifest.theme
+        packages.push(manifest) if manifest.theme and not (options.argv.enabled and @isPackageDisabled(child))
```

Comment:
I think these inline ifs are getting a bit hard to understand at a glance.  Would you mind converting them into regular ifs? Eg
```coffee
if mainfest.theme and not (options.argv.enabled and @isPackageDisabled(manifest.name))
```
Also, `manifest.name` is clearer in its intent than just `child`.